packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.styles.ts

```diff
@@ -10,7 +10,8 @@ const GlobalClassNames = {
   innerField: 'ms-ChoiceField-innerField',
   imageWrapper: 'ms-ChoiceField-imageWrapper',
   iconWrapper: 'ms-ChoiceField-iconWrapper',
-  labelWrapper: 'ms-ChoiceField-labelWrapper'
+  labelWrapper: 'ms-ChoiceField-labelWrapper',
+  checked: 'is-checked'
```

Comment:
Wouldn't we want something that follows the other classnames in here like:

```ts
checked: ms-ChoiceField-checked
```

or

```ts
isChecked: ms-ChoiceField-isChecked
```

This `checked` or `isChecked` property should also be added to the `IChoiceGroupOptionStyles` interface in `ChoiceGroupOption.types.ts` since we want `GlobalClassNames` to have parity with it.

Reply:
Actually, looking through the rest of the code I'm wondering, why do we need this in here? It doesn't seem like this is actively being used for something.

Also, it is kind of counter-intuitive to have this in here as the `GlobalClassNames` refer to specific parts of the component, as it can be seen if you check the other classnames like `innerField` and `imageWrapper`. The proposed `checked` addition would be much more in line if added as a property to `IChoiceGroupOptionProps`.

Reply:
`is-checked` was there in fabric 5, so if you relied on it and upgraded to fabric 6, your customizations would break